From 501ea14ca935452a1fff078206b976cb047c0e67 Mon Sep 17 00:00:00 2001
From: Mianhan Liu <liumh1@shanghaitech.edu.cn>
Date: Thu, 30 Sep 2021 22:48:52 +0800
Subject: [PATCH] net/802/garp.c: remove superfluous headers

garp.c hasn't use any macro or function declared in linux/etherdevice.h and linux/netdevice.h.
Thus, these files can be removed from garp.c safely without
affecting the compilation of the ./net module
---
 net/802/garp.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/net/802/garp.c b/net/802/garp.c
index f6012f8e5..3a7197d43 100644
--- a/net/802/garp.c
+++ b/net/802/garp.c
@@ -7,8 +7,6 @@
 #include <linux/kernel.h>
 #include <linux/timer.h>
 #include <linux/skbuff.h>
-#include <linux/netdevice.h>
-#include <linux/etherdevice.h>
 #include <linux/rtnetlink.h>
 #include <linux/llc.h>
 #include <linux/slab.h>
-- 
2.25.1

